webapi/src/main/scala/org/knora/webapi/routing/v2/ResourcesRouteV2.scala

```diff
@@ -176,16 +177,43 @@ class ResourcesRouteV2(routeData: KnoraRouteData) extends KnoraRoute(routeData)
                         } yield requestMessage
 
                         RouteUtilV2.runRdfRouteWithFuture(
-                            requestMessageFuture,
-                            requestContext,
-                            settings,
-                            responderManager,
-                            log,
-                            ApiV2WithValueObjects
+                            requestMessageF = requestMessageFuture,
+                            requestContext = requestContext,
+                            settings = settings,
+                            responderManager = responderManager,
+                            log = log,
+                            responseSchema = ApiV2WithValueObjects
                         )
                     }
                 }
             }
+        } ~ path("v2" / "resources" / "history" / Segment) { resourceIriStr: IRI =>
```

Comment:
I would personally prefer the following:

```
/v2/resources/[resourceIRI]/history
```

Reply:
This would conflict with the existing route:

```
/v2/resources/[resourceIRI]/[resourceIRI]
```